### PR TITLE
chore: 🤖 remove oidc-crud feature flag

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
✅ Closes: ICU-5597

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-5597)

### CODE REVIEW

**Review by cameronperera — APPROVED** [review]

File: ui/admin/app/templates/scopes/scope/auth-methods/index.hbs [hunk 1]
```diff
@@ -119,35 +119,15 @@
           {{#each @model as |authMethod|}}
             <body.row as |row|>
               <row.headerCell>
-                {{!
-                  Show link to auth method route if non-OIDC and when
-                  oidc-crud feature is enabled, otherwise show just the name
-                  (when the method is OIDC and oidc-crud is disabled)
-                }}
-                {{#if authMethod.isOIDC}}
```
Comment (gsusmi) [inline ui/admin/app/templates/scopes/scope/auth-methods/index.hbs:127]: Isn't this check still needed?

**Review by gsusmi — CHANGES_REQUESTED** [review]

File: ui/admin/app/templates/scopes/scope/auth-methods/index.hbs (hunk 1 above)
Comment (cameronperera) [reply ui/admin/app/templates/scopes/scope/auth-methods/index.hbs:127]: Yes it is! Good catch.

Comment (cameronperera) [reply ui/admin/app/templates/scopes/scope/auth-methods/index.hbs:127]: Actually, I am not sure now. I checked out the branch compared to main and they both function the same way. I believe we do want to show OIDC auth methods. I am fairly certain that was been in production for a while. @gsusmi or @randallmorey do you have more input for this.

Comment (gsusmi) [reply ui/admin/app/templates/scopes/scope/auth-methods/index.hbs:127]: Looking through this again, other than checking for capability to read an auth-method, auth-method type checks shouldn't be needed. Sorry for causing the confusion.

**Review by gsusmi — APPROVED** [review]

File: ui/admin/app/templates/scopes/scope/auth-methods/index.hbs (hunk 1 above)
Comment (lisbet-alvarez) [reply ui/admin/app/templates/scopes/scope/auth-methods/index.hbs:127]: No worries!!

**Pull request merged**